test/parallel/test-tls-inception.js

```diff
@@ -59,11 +61,13 @@ a.listen(common.PORT, function() {
       rejectUnauthorized: false
     });
     ssl.setEncoding('utf8');
+    var buf = '';
     ssl.once('data', function(data) {
-      assert.equal('hello', data);
+      buf += data;
       gotHello = true;
     });
```

Comment:
I said about `gotHello` because would the test exit if no `end` event was received?
Yes, I can send a PR.